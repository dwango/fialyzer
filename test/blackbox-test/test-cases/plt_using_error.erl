-module(plt_using_error).

-export([main/0]).

main() ->
    {{Year, Month, Day}, {Hour, hoge, Second}} = calendar:local_time(),
    0.
