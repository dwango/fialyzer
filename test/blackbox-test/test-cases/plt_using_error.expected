TODO:filename:-1: Type error: type mismatch;
  found   : {{any(), any(), any()}, {any(), 'hoge', any()}}
  required: {{number(), number(), number()}, {number(), number(), number()}}
there is no solution that satisfies subtype constraints
